crates/sqlexec/src/planner/logical_plan/alter_database_rename.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Clone, Debug)]
```

Comment:
i know these are rather empty right now, but they will fill up with all of the boilerplate for (de)serialization & wanted to keep things a bit more organized.